CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- **CUMULUS-548** - Create a Lambda task that generates EMS distribution reports
+  - Updates required by this change:
+    1. Create an S3 bucket to store S3 Server Access logs.
+    2. Create an S3 bucket to store EMS Distribution reports.
```

Comment:
Is there any reason we can't use system/internal bucket?  That way we don't need those configurations.
The EMS Distribution reports should be stored in the same place as the other EMS reports (Ingest,Archive, Delete).